Multisig/UI/App/NavigationRouter.swift

```diff
@@ -24,6 +25,43 @@ extension NavigationRouter {
     }
 }
 
+class CompositeNavigationRouter: NavigationRouter {
+    static let shared = CompositeNavigationRouter(routers: [ExtendedNavigationRouter(), DefaultNavigationRouter()])
```

Comment:
Do we really need this differentiate between Routers? can we just merge them all ?

Reply:
it was easier to handle different web links versions - I just added the new routes and it works with the previous way, too, leading to less complicated classes

Reply:
in the future, when the old way is not supported, then we just remove that router